test/integration/notice_test.rb

```diff
@@ -0,0 +1,30 @@
+require 'test_helper'
+
+class NoticeGovspeakTest < ActionDispatch::IntegrationTest
```

Comment:
I think we should put this in a 'components' directory so it's not confused with the other template level integration tests.